app/Http/Controllers/V2/Dashboard/ActiveProjectsTableController.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+namespace App\Http\Controllers\V2\Dashboard;
+
+use App\Helpers\TerrafundDashboardQueryHelper;
+use App\Http\Controllers\Controller;
+use App\Models\V2\Forms\FormOptionList;
+use App\Models\V2\Forms\FormOptionListOption;
+use App\Models\V2\Projects\Project;
+use App\Models\V2\Projects\ProjectReport;
+use App\Models\V2\Sites\SiteReport;
+use App\Models\V2\TreeSpecies\TreeSpecies;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\DB;
+
+class ActiveProjectsTableController extends Controller
+{
+    public function __invoke(Request $request)
+    {
+        return response()->json([
+            'project_list' => $this->getAllProjects($request),
+        ]);
+    }
+
+    public function getAllProjects($request)
+    {
+        $projects = Project::query();
+        $projects = TerrafundDashboardQueryHelper::buildQueryFromRequest($projects, $request)
+            ->pluck('id')->toArray();
+        $activeProjects = [];
+        foreach ($projects as $projectId) {
```

Comment:
This controller is issuing a _ton_ of superfluous DB queries. You're pulling the ids for a bunch of projects and then iterating on those ids and in a lot of cases pulling the whole record, or part of the record from the DB in the code below. This is the definition of N+1 queries, and this kind of pattern is one of the main reasons the terramatch BE is so slow today. 

Two ideas:
* Just pull all the records initially instead of just the IDs. Then at least you're skipping most of the additional queries below.
* If you feel this really is the best pattern, then the additional queries should be done in bulk, which is a much more substantial refactor of this method. I honestly wouldn't do it this way because it's complicated and error-prone, but you could, for instance, pull the list of all names in one query with `Project::whereIn('id', $projects)->pluck('name')` instead of going back to the database for every single id in the initial pull.